mod/wikirate_source/features/source.feature

```diff
@@ -0,0 +1,36 @@
+@javascript
+Feature: source
+  In order to have source to support notes/metric values
+  As a User
+  I want to create a source
+
+  Background:
+    Given I am signed in as Joe Admin 
+
+  Scenario: Create a link source
+    When I wait until ajax response done
+    And I go to new Source
+    And I fill in "card_subcards__Link_content" with "http://example.com"
+    And I fill in "card_subcards__Title_content" with "a test link source"
```

Comment:
nice test.  We need to work on these "fill in" steps; cukes aren't supposed to have code identifiers.  Maybe we need to add labels or something?